cheta/converters.py

```diff
@@ -489,16 +513,25 @@ def tel2eng(dat):
     # 4OAVOBAT is modified by both patches since it is an average of MSIDs in both parts of the
     # patch. Use the second time value as this is when the process is complete. See obc4eng() for
     # both times and further details.
-    patch_time = DateTime("2014:342:16:32:45").secs
+    patch_times = {
+        "pr-361": DateTime("2014:342:16:32:45").secs,
+        "pr-575": DateTime("2024:102:11:20:00").secs,
+    }
+
+    msid = {
+        "pr-361": "4OAVOBAT",
+        "pr-575": "4OAVHRMT",
```

Comment:
I'm just guessing that  4OAVHRMT one needs the wide mode treatment at this point.

Reply:
@matthewdahmer or @christian-anderson, should 4OAVHRMT in the cheta cxc archive be switched to be the from-WIDE values after PR-575?  I wasn't sure what it was an average over, but saw that it was read out and checked as part of the CAP.